src/network/peloton_server.cpp

```diff
@@ -120,7 +120,7 @@ void PelotonServer::SSLInit() {
   // load trusted CA certificates (peer authentication)
   if (SSL_CTX_load_verify_locations(ssl_context, certificate_file_.c_str(),
                                     nullptr) != 1) {
-    LOG_ERROR("Exception when loading root_crt!");
+    LOG_WARN("Exception when loading root_crt!");
```

Comment:
Do we want to add a more descriptive message here to help people diagnose the problem?

Reply:
Yes, the message could be more helpful. Let me do that in my next PR... more clean up to do.